### PR TITLE
feat(tutorial): complete step 2

### PR DESCRIPTION
**Pull request by rlgarret**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

**Comment by rlgarret** [comment]
Are you saying that at the time of pull request #1083, the state of the files, being at end of tutorial-2, does not match expectations?  Or does the current state of the rendered webpage, which for me is now at the end of tutorial-3 does not match expectations?  Do I need to compare my source files character by character against a standard?  Thx.

**Comment by kellychurchill** [comment]
The build preview for your pull request can be found here:
 https://deploy-preview-1083--carbon-tutorial.netlify.com/#/repos 

And looks like this:
Screen Shot 2019-06-28 at 2 17 10 PM

This does not match the expected outcome from the tutorial: https://react-step-3--carbon-tutorial.netlify.com/#/repos

Thanks!

**Comment by rlgarret** [comment]
The root cause is my fork has not pulled the changes made to the base after the tutorial sessions began. I am 2 commits behind and cannot figure out how to git those updates into my fork.  I can paste in the specific block from src/index.scss that creates the border around the repo table.  But  several other changes will still be missing.  Perhaps those are immaterial.  I missed the guidance on how to stay in sync.  Should I compare each tutorial step's branch separately and create myself a pull request into my branch?  I tried that with master but it did not resolve my problem with step-2.  Or is there some easier way? Thx.

File: src/content/RepoPage/RepoPage.js [hunk 1]
```diff
@@ -1,7 +1,70 @@
...
+    stars: '456',
+    links: 'Links',
+  },
+];
 
 const RepoPage = () => {
-  return <div>REPO PAGE</div>;
+  return (
+    <div>
+      REPO PAGE
```
Comment (kellychurchill) [inline src/content/RepoPage/RepoPage.js]: `REPO PAGE` should be removed.

**Comment by rlgarret** [comment]
@kellychurchill    Ah!  I see.  I was completely hung up on the right and left margins being off because I am missing all the updates made to the tutorial base.  The copy/paste error has been fixed.  Thx.

Screen Shot 2019-07-28 at 7 12 22 AM

**Pull request closed without merging**